client/src/Login/Login.js

```diff
@@ -35,6 +50,48 @@ export function Login() {
     history.push('/dashboard')
   }
 
+  const contactUs = ({ message }) => {
+    return `${message} <a href="mailto:tech@pieforproviders.com">${t(
+      'contactUs'
+    )}</a>{' '} ${t('forSupport')}`
+  }
+
+  const ResendToken = () => {
+    return (
+      <div>
+        {`${t('yourConfirmationToken')} ${t(apiError?.type)}. ${t(
+          'mustUseConfirmationEmail'
+        )} ${t('requestNewConfirmation')} `}
+        <a href="www.google.com" target="_blank">
+          Placeholder Link
+        </a>
+      </div>
+    )
+  }
+
+  const errorMessage = ({ attribute, type }) => {
+    switch (true) {
```

Comment:
We might want to use the `attribute` here.

```
switch(attribute) {
  case 'email': {
   // ...
  }
}
```

Or we could refactor it a bit. If you prefer to do it in a different PR, no worries.

```js
const errorMessages = {
  email: {
    already_confirmed: () => t('alreadyConfirmed'),
    confirmation_period_expired: () => t('confirmationPeriodExpired'),
    default: () => contactUs({ message: t('genericEmailConfirmationError') })
  },
  confirmation_token: {
    blank: () => <ResendToken type="blank" />,
    invalid: () => <ResendToken type="invalid" />,
    default: () => contactUs({ message: t('genericEmailConfirmationError') })
  },
  default: () => t('genericConfirmationError'),
}

const attributeError = errorMessages[attribute]
if (!attributeError) {
  return errorMessages.default()
}

const fn = attributeError[type] || attributeError.default
return fn()
```